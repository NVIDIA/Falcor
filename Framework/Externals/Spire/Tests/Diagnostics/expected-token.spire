// expected one token, but got another

int foo()
{
	int a = 3 ]
}