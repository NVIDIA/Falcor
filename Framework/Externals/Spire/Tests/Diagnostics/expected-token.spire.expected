result code = -1
standard error = {
Tests/Diagnostics/expected-token.spire(5): error 20001: unexpected ']', expected ';'
}
standard output = {
}
